runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -902,8 +924,11 @@ MM_CopyForwardScheme::reserveMemoryForObject(MM_EnvironmentVLHGC *env, UDATA com
 	if (NULL != result) {
 		regionList->_cacheAcquireCount += 1;
 		regionList->_cacheAcquireBytes += allocDescription.getBytesRequested();
+		setCompressedSurvivorCards(env, (void*)result, (void*) ((UDATA)result + allocDescription.getBytesRequested()));
+		/* need to set only 1 card for the object */
+//		setCompressedSurvivorCards(env, (void*)result, (void*) ((UDATA)result + (1 << J9VMGC_SIZECLASSES_LOG_SMALLEST)));
```

Comment:
I remember you said this did not work, but not understood why? anyhow, let's remove this

Reply:
yes,this can cause occasionally crash, looks like there is a change that we allocate one memory for multiple objects( not TLH case).